.gitignore

```diff
@@ -8,6 +8,12 @@ syntax: glob
 *.userosscache
 *.sln.docstates
 
+# Mac/OS X-specific files
+.DS_Store
+
+# Visual Studio 2015 cache/options directory
+.vs/
+
```

Comment:
Added some additional directories that are created by OS X (when working via parallels) and the newer versions of Visual Studio